### PR TITLE
Remove tab header search shortcut

### PR DESCRIPTION
**Pull request by danieltucunduva**
Associated Issue: #4110

### Summary of Changes

* Remove the tab header search shortcut

image

### CODE REVIEW

**Comment by darkwing** [comment]
Nice work @danieltucunduva!  I'm seeing the lines slightly off between the two panes:

offbypixel

I changed `.source-outline-tabs .tab`'s `padding` to `8px 8px 7px 8px` and everything lines up.  Could you try that?

File: src/components/PrimaryPanes/index.js [hunk 1]
```diff
@@ -135,6 +134,8 @@ class PrimaryPanes extends Component {
   }
 }
 
+PrimaryPanes.displayName = "PrimaryPanes";
```
Comment (nyrosmith) [inline src/components/PrimaryPanes/index.js]: Small nit: we no longer need displaynames

File: src/components/PrimaryPanes/index.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ class PrimaryPanes extends Component {
   }
 
   renderHeader() {
```
Comment (nyrosmith) [inline src/components/PrimaryPanes/index.js]: You can remove the entire function

**Comment by danieltucunduva** [comment]
@darkwing thanks for pointing it out
The original commit looked perfect here?
In any case, I will replace the values with the ones you suggested and commit if they work here too.

File: src/components/PrimaryPanes/index.js (hunk 1 above)
Comment (danieltucunduva) [reply src/components/PrimaryPanes/index.js]: Corrected

**Pull request merged**